wildfly-getting-started-archetype/src/main/resources/archetype-resources/src/main/webapp/index.html

```diff
@@ -0,0 +1,79 @@
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright (c) 2023, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+<!doctype html>
+<html class="no-js" lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title>Welcome to WildFly Application Server</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="stylesheet" href="normalize.min.css">
+    <link rel="stylesheet" href="wildfly.css">
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon"/>
+    <script>
+        function submitClicked() {        	
+        	var name = document.getElementById('nameInput').value;
+            name = escape(name);
+            var request = new XMLHttpRequest();
+            var result = document.getElementById('result');             
+            request.onreadystatechange=function() {
+                if (request.readyState == 4) {
+                    if (request.status == 200) {
+                        result.innerText = request.responseText;
+                    } else {
+                        result.innerText = 'Unexpected response ' + request.status;
+                    }
+                }
+            }
+
```

Comment:
This is very minor, but there are better ways to do this in JavaScript now. I can update this with a PR though if you'd like.

Reply:
Please do if you don't mind - we should be aiming to be as correct as we can.